src/mscorlib/corefx/System/Globalization/TextInfo.Unix.cs

```diff
@@ -10,7 +10,13 @@ namespace System.Globalization
 {
     public partial class TextInfo
     {
-        private bool m_needsTurkishCasing;
+        enum TurkishCasing
```

Comment:
Could this be `enum TurkishCasing : byte`, to keep the field size the same as before?